src/mom5/ocean_param/vertical/ocean_vert_kpp_mom4p1.F90

```diff
@@ -941,7 +967,8 @@ end subroutine ocean_vert_kpp_mom4p1_init
 ! </DESCRIPTION>
 !
 subroutine vert_mix_kpp_mom4p1 (aidif, Time, Thickness, Velocity, T_prog, T_diag, Dens, &
-                                swflx, sw_frac_zt, pme, river, visc_cbu, diff_cbt, hblt_depth, do_wave)
+!dhb599:                                swflx, sw_frac_zt, pme, river, visc_cbu, diff_cbt, hblt_depth, do_wave)
```

Comment:
Ditto here.. in fact, `vert_mix_jpp_mom4p1` looks like a whole new function...?

Reply:
Ok, it looks like there are some pretty important changes here. Mixing due to Langmuir circulation has been added. It's related to work I'm doing. I'm not happy with how it interfaces with the rest of the code though.